### PR TITLE
Adding predefined templates to UI

### PR DESCRIPTION
**Pull request by nitinmotgi**

### CODE REVIEW

File: cdap-ui/templates/apps/predefined/DatabaseToCassandra.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            }
+        },
+        "connections": [
+            {
+                "source": "Database-batchsource-1442850877586",
+                "target": "Cassandra-batchsink-1442851032454"
+            }
+        ]
+    }
+}
```
Comment (bdmogal) [inline cdap-ui/templates/apps/predefined/DatabaseToCassandra.json]: missing newline

File: cdap-ui/templates/apps/predefined/KafkaToCassandra.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            }
+        },
+        "connections": [
+            {
+                "source": "Kafka-realtimesource-1442764347297",
+                "target": "Cassandra-realtimesink-1442764340811"
+            }
+        ]
+    }
+}
```
Comment (bdmogal) [inline cdap-ui/templates/apps/predefined/KafkaToCassandra.json]: missing newline

File: cdap-ui/templates/apps/predefined/KafkaToCassandraAndCDAPStream.json [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                "source": "Kafka-realtimesource-1442764512341",
+                "target": "Cassandra-realtimesink-1442764340811"
+            },
+            {
+                "source": "Kafka-realtimesource-1442764512341",
+                "target": "Stream-realtimesink-1442764524404"
+            }
+        ]
+    }
+}
```
Comment (bdmogal) [inline cdap-ui/templates/apps/predefined/KafkaToCassandraAndCDAPStream.json]: missing newline

File: cdap-ui/templates/apps/predefined/TwitterToHBase.json [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+            }
+        },
+        "connections": [
+            {
+                "source": "Twitter-source-1",
+                "target": "Table-sink-4"
+            }
+        ]
+    }
+}
```
Comment (bdmogal) [inline cdap-ui/templates/apps/predefined/TwitterToHBase.json]: missing newline

**Comment by bdmogal** [comment]
4-5 missing newlines at the end of files, which may cause build failures. Other than that, presume tested. LGTM!

File: cdap-ui/templates/apps/predefined/DatabaseToCassandra.json (hunk 1 above)
Comment (nitinmotgi) [inline cdap-ui/templates/apps/predefined/DatabaseToCassandra.json]: fixed.

**Pull request merged**